graphene_django/range_types.py

```diff
@@ -0,0 +1,29 @@
+from graphene import ObjectType
+from graphene import (
+    Float,
+    Int,
+    DateTime,
+    Date
+)
+
+class RangeResolver:
+    def resolve_lower(parent, info):
+        return parent.lower
+    def resolve_upper(parent, info):
+        return parent.upper
+
+class DateTimeRangeType(RangeResolver, ObjectType):
+    lower = DateTime()
+    upper = DateTime()
```

Comment:
Are these fields `required`?

Reply:
Not quite, I think it could be optional.

One problem is the original method which returns a tuple for (lower, upper), it is working for IntegerRange which is tested in test cases, but it is not working for DateTimeRange.